manga-finder/src/components/Card.tsx

```diff
@@ -3,27 +3,18 @@ import TagSquare from "./TagSquare";
 
 function getImgUrl(title: string, id: string) {
   let url: string = "https://i.imgur.com/hTmBaJL.jpeg";
-  if (title === "Alice") {
-    url = "https://i.imgur.com/UsfSZvM.jpeg";
-  }
   return url;
 }
 
 const MangaCard = (props: IMangaCard) => {
   const manga = props;
   //loop that makes a new TagSquare foreach manga.tag
-  manga.tags.forEach((tag) => {
-    <TagSquare
-      id={tag.id}
-      type={tag.type}
-      name={tag.name}></TagSquare>;
-  });
   return (
     <a
       href="#"
-      className="mangacard rounded-lg w-24 h-36 bg-slate-100 m-4 relative">
+      className="mangacard rounded-lg bg-slate-100 m-4 relative w-24 h-36 xl:w-48 xl:h-72 sm:h-48 sm:w-32">
       <img
-        className="mangacover rounded-lg z-1 w-24 h-36"
+        className="mangacover rounded-lg z-1 w-full h-full"
         src={manga.imageUrl}
         alt={manga.title + "cover"}></img>
```

Comment:
could use template strings

Reply:
Thank you Mish, I'll implement those fixes in my latest styling updates.